messages/project.deploy.org.md

```diff
@@ -0,0 +1,197 @@
+# summary
+  
+Deploy source to an org.
```

Comment:
Maybe "source and metadata" so testers aren't looking for a separate metadata deployment command?

Reply:
yep